src/engine/instrument.h

```diff
@@ -880,6 +881,63 @@ struct DivInstrument {
   DivInstrumentPowerNoise powernoise;
   DivInstrumentSID2 sid2;
 
+  DivInstrumentPOD() :
+    type(DIV_INS_FM) {
+  }
+};
+
+struct MemPatch {
+  MemPatch() :
+    data(NULL)
+    , offset(0)
+    , size(0) {
+  }
+
+  ~MemPatch() {
+    if (data) {
+      delete[] data;
```

Comment:
Set data to NULL after this just to be sure.

Reply:
👍 set it to null